quiche/src/crypto.rs

```diff
@@ -499,22 +499,7 @@ fn derive_initial_secret(secret: &[u8], version: u32) -> hkdf::Prk {
         0xa4, 0xc8, 0x0c, 0xad, 0xcc, 0xbb, 0x7f, 0x0a,
     ];
 
-    const INITIAL_SALT_DRAFT29: [u8; 20] = [
-        0xaf, 0xbf, 0xec, 0x28, 0x99, 0x93, 0xd2, 0x4c, 0x9e, 0x97, 0x86, 0xf1,
-        0x9c, 0x61, 0x11, 0xe0, 0x43, 0x90, 0xa8, 0x99,
-    ];
-
-    const INITIAL_SALT_DRAFT27: [u8; 20] = [
-        0xc3, 0xee, 0xf7, 0x12, 0xc7, 0x2e, 0xbb, 0x5a, 0x11, 0xa7, 0xd2, 0x43,
-        0x2b, 0xb4, 0x63, 0x65, 0xbe, 0xf9, 0xf5, 0x02,
-    ];
-
     let salt = match version {
```

Comment:
lets mute the warning for now? We might want to support new versions in future ...